src/components/resumes/Details.jsx

```diff
@@ -56,6 +80,15 @@ function Details() {
     }
   };
 
+  const getSkills = async (event) => {
+    console.log("Skill input: ", skillInput);
+    event.preventDefault();
+    const response = await getSkillsByName(skillInput);
+    console.log(response);
+  };
```

Comment:
you should delete console.log in line 84 and 87

Reply:
Done